### PR TITLE
Test-Vectors for Lotus v0.10.0

### PR DESCRIPTION
**Pull request by Alexey-N-Chernyshov**
### Description of the Change
Test Vectors (https://github.com/filecoin-project/test-vectors) tested and fixed.
577 tests passed.
26 broken tests are ignored (https://github.com/filecoin-project/test-vectors#brokenincorrect-vectors).
23 tests with chaos actor are ignored (https://github.com/filecoin-project/test-vectors#special-test-harness-actor).

### Usage Examples or Tests *[optional]*
See testvectors README.md

### CODE REVIEW

File: core/vm/version.hpp [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  /**
+   * Enumeration of network upgrades where actor behaviour can change
+   */
+  enum class NetworkVersion {
+    kVersion0,
+    kVersion1,
+    kVersion2,
+    kVersion3,
+    kVersion4,
+    kVersion5
```
Comment (turuslan) [inline core/vm/version.hpp:23]:
```suggestion
    kVersion5,
```

File: test/core/test_vectors/test.cpp [hunk 2]
```diff
@@ -119,15 +270,45 @@ TEST_P(TestVectors, Vector) {
...
+TEST_P(TestVectors, Vector) {
+  auto &mv{GetParam()};
+  fc::vm::actor::cgo::test_vectors = true;
+  fc::vm::actor::cgo::config(
+      1 << 20,
+      UINT64_C(10) << 40,
+      {fc::primitives::sector::RegisteredProof::StackedDRG32GiBSeal,
+       fc::primitives::sector::RegisteredProof::StackedDRG64GiBSeal});
+
+  std::cout << "Test path: " << mv.path << std::endl;
```
Comment (turuslan) [inline test/core/test_vectors/test.cpp:282]: Please remove, it can be found in test case name

**Review by turuslan — APPROVED** [review]

File: deps/our-filecoin-ffi/install-filcrypto.sh [hunk 3]
```diff
@@ -6,7 +6,7 @@ set -Eeo pipefail
 ffi_dir="../filecoin-ffi"
 release_sha1=$(cd $ffi_dir && git rev-parse HEAD)
 if [ -e filecoin_ffi_commit_installed ] && [ "$(cat filecoin_ffi_commit_installed)" = "$release_sha1" ]; then
-  (>&2 echo "filecoin-ffi is already installed")
+  # (>&2 echo "filecoin-ffi is already installed")
```
Comment (artyom-yurin) [inline deps/our-filecoin-ffi/install-filcrypto.sh:9]: Why?

Comment (artyom-yurin) [reply deps/our-filecoin-ffi/install-filcrypto.sh:9]: maybe remove

**Review by artyom-yurin — APPROVED** [review]

**Pull request merged**